pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -105,7 +105,7 @@ const (
 	Version1_15 = "1.15"
 
 	// DefaultVersion represents default Kubernetes version supported by EKS
-	DefaultVersion = Version1_14
+	DefaultVersion = Version1_15
 
 	// LatestVersion represents latest Kubernetes version supported by EKS
 	LatestVersion = Version1_14
```

Comment:
do we need to update this value as well ?

Reply:
oh, this should have changed when we released 1.15. Thanks for the catch! Fixed